src/Domain/Command/CreateArticle.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Domain\Command;
+
+class CreateArticle
```

Comment:
Du coup, si on voulait être au maximum cohérent avec la terminologie utilisée dans l’entité `Article`, peut-être qu’il faudrait appeler la commande (ainsi que le handler) `WriteAnArticle` ou quelque chose comme ça?

(Ce que je dis là peut plus ou moins renvoyer à la notion d’ubiquitous language en DDD, qui est en gros la terminologie que le P.O. et les stakeholders utilisent pour parler de leur métier, et que nous devons utiliser également de manière systématique dans le code, non sans avoir préalablement traduit la dite terminologie en anglais si ce n’est pas déjà le cas, et s’être assurés de son homogénéité, par exemple en n’utilisant qu’un seul et même mot pour un seul et même concept, en écartant de possibles synonymes.)

Reply:
:100: